Standard Library/List/task.md

```diff
@@ -90,4 +89,4 @@ This idea describes *Insertion Sort*:
         case y :: ys => insert(y, insertionSort(ys))
       }
 
-Complete the definition insertion sort by filling in the blanks in the definition below.
+Complete the definition by filling in the blanks.
```

Comment:
Maybe we can keep the insertion sort mention here for a student to create an association right away?